Blish HUD/_Utils/ProcessUtil.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Blish_HUD._Utils {
```

Comment:
I apologize as we don't have this really written here.  Any of the directories that are prefixed with `_` are intended to not be part of the namespace.  That is to say, the utils would be placed directly into the Blish_HUD namespace.  With this being pretty specific in purpose we can probably mark it internal or move it to a namespace under gameservices next to where it's used as a internal util there.

Reply:
Updated the PR to make the class internal & removed the _Utils namespace